course/src/main/java/org/openedx/course/data/repository/CourseRepository.kt

```diff
@@ -51,7 +51,7 @@ class CourseRepository(
     suspend fun preloadCourseStructure(courseId: String) {
         val response = api.getCourseStructure(
             "stale-if-error=0",
-            "v1",
+            "v3",
```

Comment:
we can define it config.yaml

Reply:
Same, improvements with versions in a separate PR?